packages/importer/src/columns.ts

```diff
@@ -149,23 +153,80 @@ export const coerceValue = (
         ? { value: parseMultiple(String(value)), isError: false }
         : { value: null, isError: true };
     }
+    case 'file': {
+      const res = await parseFile((value as string).trim(), proxyFunction);
+      if (!res) return { value: null, isError: true };
+      return {
+        value: { name: 'upload', mediaType: res.mediaType, base64Content: res.base64Content } as XataFile,
+        isError: false
+      };
+    }
+    case 'file[]': {
+      // Cannot import multiple raw Base64 files because we will split in the wrong place
+      const promises = (value as string).split(/[,;|]/).map((uri) => parseFile(uri, proxyFunction));
+      const files = await Promise.all(promises);
+      const isError = files.some((file) => file === null);
+      const formatted = files.map(
+        (file) => ({ name: 'upload', mediaType: file?.mediaType, base64Content: file?.base64Content } as XataFile)
```

Comment:
Just setting the name as 'upload' here since I don't think there's a reliable way to get the name? We also error in the backend when the file name has invalid characters.